packages/xchain-polkadot/src/client.ts

```diff
@@ -284,7 +284,7 @@ class Client implements PolkadotClient, XChainClient {
 
       const account = response.data
 
-      return account && (!asset || assetToString(asset) === assetToString(AssetDOT))
+      return account && (!assets || assets.findIndex((asset) => assetToString(AssetDOT) === assetToString(asset)) >= 0)
```

Comment:
As mentioned before: `findIndex`[ is not supported in IE](https://caniuse.com/?search=findIndex).